python/pyproject.toml

```
@@ -87,10 +84,7 @@ torch25-cpu = [
     # We currently don't suppoer MAC environment as a result.
```

```suggestion
    # We currently don't support MAC environment as a result.
```

Also is this still true?